lambdas/initialise-ipv-session/src/test/java/uk/gov/di/ipv/core/initialiseipvsession/InitialiseIpvSessionHandlerTest.java

```diff
@@ -638,7 +684,7 @@ void shouldValidateAndStoreAnyInheritedIdentityWhenNoInheritedVcExist() throws E
     }
 
     @Test
-    void shouldHandleParseExceptionFromCheckingInheritedIdentityVotStrength() throws Exception {
+    void shouldHandleParseExceptionFromSendingAuditEvent() throws Exception {
```

Comment:
Think the name of this could be something else now as we're no longer throwing a ParseException